.github/workflows/build-and-push-demo-pipeline.yaml

```diff
@@ -2,7 +2,7 @@ name: Build and push demo pipeline image
 
 on:
   schedule:
-    - cron: '0 4 * * *' # 04:00 AM UTC every day
+    - cron: '0 2 * * *' # 02:00 AM UTC every day
```

Comment:
Hi @hoangtungdinh , just one question, is it now needed to perform the demo pipeline build everyday? Or is it not possible to set the trigger just when there are new changes?

Reply:
Hi @AsamDiegoSanchez,

The reason we build the demo pipeline periodically is that the demo pipeline depends on several other repositories. Triggering the demo pipeline based on changes in those repositories requires more development effort and introduces some dependencies between the repositories.

For now, since development is stable, we can build the demo pipeline on a weekly or bi-weekly basis. In the future, when we have a Python result pooling and Python text report, we can remove the Docker-based demo pipeline and replace it with a Python-based pipeline that does not require the periodic build.